index.js

```diff
@@ -427,3 +427,19 @@ module.exports.sync = (command, args, options) => {
 };
 
 module.exports.shellSync = (command, options) => handleShell(execa.sync, command, options);
+
+module.exports.fork = (filePath, args, options) => {
+	const stdioOption = stdio.fork(options);
+	delete options.stdin;
+	delete options.stdout;
+	delete options.stderr;
```

Comment:
I think we should avoid mutating caller's arguments. This is a side-effect the caller might not expect.

For example, let's say you're calling `fork()` twice in a row using the same `options` object. The second call will get different `options` with the current code.